goship.go

```diff
@@ -760,23 +777,210 @@ func extractOutputHandler(fn func(http.ResponseWriter, *http.Request, string, st
 	}
 }
 
+type User struct {
+	UserName, UserAvatar string
+	auth                 auth
+}
+
+func getUser(r *http.Request) (User, error) {
+	u := User{}
+	session, err := store.Get(r, session_name)
+	if err != nil {
+		return u, errors.New("Error Getting User Session")
+	}
+	if _, ok := session.Values["userName"]; !ok {
+		u.UserName = "genericUSER"
+		u.UserAvatar = "https://camo.githubusercontent.com/33a7d9a138ac73ece82dee977c216eb13dffc984/687474703a2f2f692e696d6775722e636f6d2f524c766b486b612e706e67"
```

Comment:
Maybe we can make the user name and avatar be exported variables in this file, so that folks extending GoShip can change these defaults if they want to.